### ISSUE TITLE
Strava Open Space Report Card

### ISSUE THREAD

**Issue by sethfitz** [OpenGlobe/myamerica-devsummit#23]
Strava has changed the way we exercise, allowing us to compete with ourselves and others using "segments."

Perhaps we should be evaluating the experiential quality of our workouts too!  By combining Strava activities with PADUS, we can show runners and cyclists the names and vital statistics of open spaces that they've passed through or near.  Athletes can compete with one another to find experience all of the parks within a city or a generalized workout area.

Through the magic of social media, we can show photos of what you experienced so you can revisit a trail run or bike ride and share it with friends.

**Comment by jmoe** [comment]
Yes yes yes. Have been excited about this idea for a long while now. Let's make it happen.

**Comment by ryanbranciforte** [comment]
+1

**Comment by stroupaloop** [comment]
+1
